doc/nrf/releases_and_maturity/releases/release-notes-3.1.0-preview1.rst

```diff
@@ -110,7 +110,14 @@ Developing with custom boards
 Security
 ========
 
-|no_changes_yet_note|
+* Updated:
+  * The Oberon PSA core to version 1.5.1 that introduces support for the following new features with the Oberon PSA driver:
+
+   * Experimental support for post-quantum cryptography schemes ML-KEM (FIPS 203), ML-DSA (FIPS 204) and XMSS verify (NIST SP 800-208).
+   * Experimental support for XChaCha20-Poly1305 AEAD algorithm.
+   * Experimental support for SHAKE128 with 256-bit output length.
+
+  * The option :kconfig:option:`CONFIG_PSA_WANT_ALG_WPA3_SAE_PT` is now removed and replaced by :kconfig:option:`CONFIG_PSA_WANT_ALG_WPA3_SAE_H2E`.
```

Comment:
```suggestion
* Updated the Oberon PSA core to version 1.5.1 that introduces support for the following new features with the Oberon PSA driver:

  * Experimental support for post-quantum cryptography schemes ML-KEM (FIPS 203), ML-DSA (FIPS 204) and XMSS verify (NIST SP 800-208).
  * Experimental support for XChaCha20-Poly1305 AEAD algorithm.
  * Experimental support for SHAKE128 with 256-bit output length.

* Removed the ``CONFIG_PSA_WANT_ALG_WPA3_SAE_PT`` Kconfig option and replaced it with :kconfig:option:`CONFIG_PSA_WANT_ALG_WPA3_SAE_H2E`.
```

Reply:
This new entry should go to release-notes-changelog, not the release notes for the already released 3.1.0 preview 1.